pages/requests/[uuid].js

```diff
@@ -40,27 +40,24 @@ const Request = () => {
   const { messages, isLoadingMessages, isMessagesError, mutateMessages, messagesData } = useMessages(uuid, session?.accessToken)
   const { files, isLoadingFiles, isFilesError, mutateFiles, filesData } = useFiles(uuid, session?.accessToken)
 
-  const [allPOs, setAllPOs] = useState(false)
+  const [allPOs, setAllPOs] = useState([])
```

Comment:
setting this by default to an empty array (which is the value the purchase orders endpoint returns where there are no po's), allows us to define `documents` in one line.